gst/nnstreamer/include/nnstreamer_cppplugin_api_filter.hh

```diff
@@ -180,6 +182,20 @@ class tensor_filter_subplugin
        Possible exceptions: ... @todo describe them!
    */
 
+  virtual void invoke_dynamic (GstTensorFilterProperties *prop,
+      const GstTensorMemory *input, GstTensorMemory *output)
+  {
+    UNUSED (prop);
+    UNUSED (input);
+    UNUSED (output);
+
+    nns_loge ("Dynamic invoke is not supported with this subplugin.");
+    assert (0);
+  }
+  /**< Optional. Invoke with dynamic output.
+   * Must set proper output info in prop.
+   */
```

Comment:
Q. Who is in charge of making flexible tensors?
`sub-plugin` or `tensor_filter_support_cc`?

Reply:
tensor_filter.c handles it like dynamic invoke of custom-easy.